src/karthik/narcissisticNumbers/index.ts

```diff
@@ -0,0 +1,27 @@
+import { checkIfNarcissticNumber } from './narcissisticNumbers';
```

Comment:
index.tx file should not contain any code except the statements to import and export functions

Reply:
fixed.